orange-stock-ios/orange-stock-ios/Entities/Stock.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  Stock.swift
+//  orange-stock-ios
+//
+//  Created by hogang on 2023/11/23.
+//
+
+import UIKit
+
+struct Stock {
+    let stockCode: String // 종목코드
+    let stockName: String // 종목명
+    let currentPrice: String // 현재가
+    let prevDayDiffSign: DiffSign // 전일 대비 부호
+    let prevDayDiffPrice: String // 전일 대비
+    let prevDayDiffRate: String // 전일 대비율
+    
+    enum DiffSign: Int {
+        case upperLimit = 1 // 상한
+        case increase       // 상승
+        case stability      // 보합
+        case lowerLimit     // 하한
+        case decrease       // 하락
+        
+        func color() -> UIColor {
+            switch self {
+            case .upperLimit, .increase:
+                return Color.stockUp
+            case .stability:
+                return Color.basic
+            case .lowerLimit, .decrease:
+                return Color.stockDown
```

Comment:
```suggestion
            case .upperLimit, .increase:
                return .stockUp
            case .stability:
                return .basic
            case .lowerLimit, .decrease:
                return .stockDown
```

타입을 충분히 예상할 수 있을 때에는 (지금의 경우엔 리턴 타입이 Fix되어 있으니까요) 
가독성을 위해 타입을 생략하는 것이 좋습니다.